documentation/source/user_section/tutorials/4/4-2.rst

```diff
@@ -0,0 +1,24 @@
+4.2 Creating a mask around the segmentation
+###########################################
+
+Next, we will create a mask to focus on the region of interest, which will increase the accuracy of the registration. Importantly, this mask is used to exclude the tissue surrounding the spinal cord, because it can move independently of the cord and negatively impact the registration.
```

Comment:
Here maybe define what a mask is? I've been suggesting a lot of definitions for jargon, we could also create a new page for jargon and just link to it?

Reply:
Given that there are likely many existing pages on the internet, maybe we could just add links for the first mention of each term?

Reply:
Links are good! Maybe down the road it would be nice to have a section in the docs with jargon, but that is a nice-to-have feature, not necessary 😃